stl/inc/deque

```diff
@@ -1571,6 +1575,18 @@ template <class _Iter, class _Alloc = allocator<_Iter_value_t<_Iter>>,
 deque(_Iter, _Iter, _Alloc = _Alloc())->deque<_Iter_value_t<_Iter>, _Alloc>;
 #endif // _HAS_CXX17
 
+#if _HAS_CXX20
+template <class _Ty, class _Alloc, class _Pr>
+void erase_if(deque<_Ty, _Alloc>& _Cont, _Pr _Pred) {
+    _Erase_remove_if(_Cont, _Pass_fn(_Pred));
+}
+
+template <class _Ty, class _Alloc, class _Uty>
+void erase(deque<_Ty, _Alloc>& _Cont, const _Uty& _Val) {

```

Comment:
We try to follow the declaration order of the synopses in the Standard absent any technical reason to do otherwise. The Standard declares `erase` before `erase_if`, and declares both of them immediate above the `pmr` bits on line 1627. (Please audit the other container headers, I assume the same comment applies to some or all of them.)

Reply:
Archaeology: This was reordered between the Library Fundamentals v2 TS and the Standard. I liked having `erase_if()` first because it's the "more fundamental" form, but Casey's right about our general convention.